### ISSUE TITLE
Update Markdown editor to the latest version of EasyMDE

### ISSUE THREAD

**Issue by aaronamm** [OrchardCMS/OrchardCore#7881]
For a WYSIWYG markdown editor, we use a fork of SimpleMDE and I think the author has renamed the project to EasyMDE.
If you navigate URL https://github.com/ionaru/simplemde-markdown-editor, you will be redirected to https://github.com/Ionaru/easy-markdown-editor.

I think we should update this script to the latest version of EasyMDE which is 2.13.0 and use it from NPM package reference 
https://www.npmjs.com/package/easymde.

Here are what I think:
- Add `easymde` as dev dependency to Node project in OrchardCore.Markdown module
- Setup Webpack, Babel, TypeScript to build a Node project
- Wrap EasyMDE as React component
- Use a wrapped EasyMDE React component in MarkdownBodyPart-Wysiwyg.Edit.cshtml and MarkdownField-Wysiwyg.Edit.cshtml
- Remove usage of Jquery, I think we should minimize using of jQuery if it is possible.

If you agree with the idea, I am happy to create a new PR for this.

However, this PR is blocked by https://github.com/OrchardCMS/OrchardCore/issues/7625 but it can bed started right away and fix a merge conflict later.

Thanks.

**Comment by agriffard** [comment]
Yes, you can create a PR.
I don't mind changing to this library because it seems to be a fork that is more maintained and has a npm repo.
Please use Assets.json and ResourceManifest (and add the version into the docs).

**Comment by agriffard** [comment]
We need of course the same plugins: Image and Shortcodes modals.

**Comment by aaronamm** [comment]
@agriffard  Thank you for your reply and suggestion. I will use Assets.json and ResourceManifest.
